examples/kafka/kafka.json

```diff
@@ -1,26 +1,26 @@
-{
-  "jobName": "kafka",
-  "image": "openpai/pai.example.kafka",
-  "taskRoles": [
-    {
-      "name": "main",
-      "taskNumber": 1,
-      "cpuNumber": 1,
-      "memoryMB": 2048,
-      "gpuNumber": 0,
-      "command": "/bin/bash /root/start.sh",
-      "portList": [
-        {
-          "label": "kafka",
-          "beginAt": 0,
-          "portNumber": 1
-        },
-        {
-          "label": "zookeeper",
-          "beginAt": 0,
-          "portNumber": 1
-        }
-      ]
-    }
-  ]
-}
+{
+  "jobName": "kafka",
+  "image": "openpai/pai.example.kafka",
```

Comment:
Why affect this file?

Reply:
this file's line ending is \r\n, so I use dos2unix to converted this file.

Reply:
ok, this is better not include at this PR, may confuse others.